binary_search_tree/Python/binary_search_tree.py

```diff
@@ -0,0 +1,74 @@
+class Leaf:
+    """
+    Class that represent a leaf node of a binary tree.
+    """
+    
+    def __init__(self, value=None, leftChild=None, rightChild=None):
+        self.value = value
+        self.leftChild = leftChild
+        self.rightChild = rightChild
+
+    def __str__(self):
+        return str(self.value)
+
+    def __repr__(self):
+        return self.__str__()
+
+    def __del__(self):
+        del self.value
+        del self.leftChild
+        del self.rightChild
+
+    def getChildren(self):
+        return [self.leftChild, self.rightChild]
+
+class BST:
+    """
+    Class that represents a Binary Search Tree
+    <https://en.wikipedia.org/wiki/Binary_search_tree>.
+    """
+    
+    def __init__(self):
+        self.root = None
+        

```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpikx6la4g/binary_search_tree/Python/binary_search_tree.py
+++ b/tmp/tmpikx6la4g/binary_search_tree/Python/binary_search_tree.py
@@ -30,7 +30,7 @@
     
     def __init__(self):
         self.root = None
-        
+
     def __del__(self):
         del self.root
 
```